pkgs/development/python-modules/pytest-astropy-header/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, astropy
+, pytest
+, pytestcov
+, codecov
+, coverage
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-astropy-header";
+  version = "0.1.2";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "astropy";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "08ly4cqxyj409cf2a8asdby1c4vdkqv4r38l0hlx4qxjkg7wfm5q";
+  };
+
+  checkPhase = "pytest";
+  checkInputs = [ pytest numpy pytestcov codecov coverage astropy ];
```

Comment:
Same comment here: I don't think that `numpy` is needed.

Reply:
Done. Thanks.